content/developer/Theme/sass_compiler.adoc

```diff
@@ -0,0 +1,400 @@
+'''
+* Change a .scss file
+* Commit just the .scss file to git
+* Change a .scss file
+* Commit just the .scss file to git
+* Compile style.scss with the php compiler
+* Commit style.css +
+
+=== Typical Workflow for the Product
+'''
+* Change a .scss file
+* Commit just the .scss file to git
+* Change a .scss file
+* Commit just the .scss file to git
+* Make a pull request. Under the "How To Test" section" add "Rebuild sass". +
+
+
+_Don't commit style.css as we do not want any more bloat added to the product. Style.css will be built during the release process._ +
+ +
+ +
+
+=== Setting Up A Compiler
+'''
+Here are a few guides on how to set up a compiler for Sass. +
+ +
+
+=== Scss PHP Compiler
+'''
+The http://leafo.net/scssphp/[Scss PHP] which used in SuiteP 7.8 or above, ensures that SuiteCRM can compile the Sass code. It also ensures that the style.css uses the same color and measurement formats. +
+
+==== Install Composer
+....
+sudo apt-get install composer
+....
+
+==== Install Scss PHP
+
+* Open a terminal
+* cd /path/to/your/instance
+* Run
+....
+composer require leafo/scssphp
+composer install
+....
+
+==== Using Scss PHP
+
+* Open a terminal
+* cd /path/to/your/instance
+* Run +
+
+*Pre 7.10*
+....
+./vendor/bin/pscss -f compressed themes/SuiteP/css/style.scss >  themes/SuiteP/css/style.css
+....
+
+*7.10 and above*
+....
+./vendor/bin/pscss -f compressed themes/SuiteP/css/Dawn/style.scss >  themes/SuiteP/css/Dawn/style.css
+./vendor/bin/pscss -f compressed themes/SuiteP/css/Day/style.scss >  themes/SuiteP/css/Day/style.css
+./vendor/bin/pscss -f compressed themes/SuiteP/css/Dusk/style.scss >  themes/SuiteP/css/Dusk/style.css
+./vendor/bin/pscss -f compressed themes/SuiteP/css/Night/style.scss >  themes/SuiteP/css/Night/style.css
+....
+
+'''
+
+=== Styling SuiteP
+'''
+The style of each feature of SuiteCRM is broken down into seperate css files. This helps to make the compiler slightly faster and it also helps to make the CSS more organised. The style.scss file is the main stylesheet file. Meaning that it imports all of the other stylesheets into style.css. The variables.scss is the stylesheet that contains all of the colors and measurements which can be easily configured. +
+ +
+
+*https://docs.suitecrm.com/community/contributing-code/coding-standards/[Please ensure that your css complies with our style guide] and http://csslint.net/[please check your code in css lint for issue before committing.]* +
+
+==== File Structure
+
+Here is a list of each section currently used. +
+
+* variables.scss
+* studio.scss
+* style.scss
+** tabs.scss
+** sidebar.scss
+** projects.scss
+** panels.scss
+** navbar.scss
+** main.scss
+** login.scss
+** listview.scss
+** forms.scss
+** editview.scss
+** detailview.scss
+** dashboard.scss
+** cases.scss
+** campaigns.scss
+** calendar.scss
+** admin.scss
+* yui.scss
+
+==== Adding New Sass Files
+
+When you need to add a new Sass file.
+
+* Create the <name>.scss in the themes/SuiteP/css/ directory.
```

Comment:
Please review throughout to add more back-ticks around file names, variable names, anything that the user might want to copy-paste.